cmd/main.go

```diff
@@ -302,18 +306,35 @@ func main() {
 		crdUtils = controllers.NewDummyCRDUtils()
 	}
 
+	gc := garbagecollector.NewGarbageCollectors(db, log.WithField("pkg", "garbage_collector"), hostApi, clusterApi, objectHandler, lead)
+
+	if Options.EnableDeregisterInactiveGC {
+
+		deregisterWorker := thread.New(
+			log.WithField("garbagecollector", "Deregister Worker"),
+			"Deregister Worker",
+			Options.DeregisterWorkerInterval,
+			gc.DeregisterInactiveClusters)
+
+		deregisterWorker.Start()
+		defer deregisterWorker.Stop()
+	}
+
+	if Options.EnableDeregisterInactiveGC {
```

Comment:
```suggestion
	if Options.EnableDeletedUnregisteredGC {
```

Looks like this check is using the wrong flag